### ISSUE TITLE
complete team page

### ISSUE THREAD

**Issue by kjappelbaum** [cheminfo/cheminfo.github.io#1]
We probably should have one. But if we have one there should be also more than one person. 

@lpatiny could you maybe send we who i should put there? I can probably copy the bio's from the Zakodium page?

**Comment by lpatiny** [comment]
You may either retrieve some bio from Zakodium page or from linkedin. Most of the people have contribution to github but some of them also worked on views in the visualizer for which activity is less visible. I'm pretty sure I forgot many people but we can complete the list later.

You may put the following people:
- Ecole polytechnique fédérale de Lausanne (Switzerland)
  - https://github.com/orgs/cheminfo/people/lpatiny
  - https://github.com/orgs/cheminfo/people/kjappelbaum
  - https://github.com/orgs/cheminfo/people/maasencioh
- Zakodium (Switzerland)
  - https://github.com/orgs/cheminfo/people/stropitek
  - https://github.com/orgs/cheminfo/people/targos
  - https://github.com/orgs/cheminfo/people/Sebastien-Ahkrin
- Univalle (Colombia, Cali)
  - https://github.com/orgs/cheminfo/people/jobo322
  - https://github.com/orgs/cheminfo/people/josoriom
  - https://github.com/orgs/cheminfo/people/andcastillo
- Universität zu Köln (Germany)
  - https://github.com/orgs/cheminfo/people/hamed-musallam
- Friedrich-Schiller-Universität Jena (Germany)
  - https://github.com/michaelwenk
  - https://github.com/jliermann
- Murdoch University (Perth, Australia)
  - https://github.com/orgs/cheminfo/people/jwist
- Johannes Gutenberg University Mainz (Germany)
  - https://github.com/jliermann
- Idorsia (Basel, Switzerland)
  - https://github.com/orgs/cheminfo/people/thsa
- LIST (Luxembourg)
  - https://github.com/ReinerD
- University of Amsterdam (Netherland)  
  - https://github.com/jdesport 
- Other
  - https://github.com/orgs/cheminfo/people/JeffersonH44
  - https://github.com/djeanner
  - https://github.com/orgs/cheminfo/people/NPellet
  - https://github.com/orgs/cheminfo/people/rufenerc
  - https://github.com/orgs/cheminfo/people/opatiny


I wonder if we can make insert some statistics for the contribution to cheminfo organisation automatically.

**Comment by kjappelbaum** [comment]
Yes, we can using the GitHub API https://docs.github.com/en/rest/reference/repos#list-repository-contributors we could put a cron job in the actions

**Comment by kjappelbaum** [comment]
but I'll make a new ticket for this as it will be a bit more involved to develop this

**Comment by kjappelbaum** [comment]
BTW, organizing it by country/organization is a nice idea!

**Comment by kjappelbaum** [comment]
some descriptions are still quite short and probably we should add more images

**Comment by lpatiny** [comment]
Should we count all the people that contribute to any of our projects ?

Maybe for the map it would make sense ?

Only for ml-matrix we have 26 contributors

I think we could combine all the contributors from mljs, image-js, cheminfo organisation and get their location

![image](https://user-images.githubusercontent.com/1484241/111022112-1fcf1280-83d1-11eb-9c71-2938ccae19cb.png)

This would then really show how large the developper base is.
